app/controllers/tweet_controller.rb

```diff
@@ -0,0 +1,73 @@
+class TweetController < ApplicationController
+
+  get '/tweets' do
+    if logged_in?(session)
+      @user = User.find_by_id(session[:user_id])
+      erb :"/tweets/index"
+    else
+      redirect '/login'
+    end
+  end
+
+  get '/tweets/new' do
+    if logged_in?(session)
+      @user = User.find_by_id(session[:user_id])
+      erb :'tweets/new'
+    else
+      redirect '/login'
+    end
+  end
+
+  post '/tweets/new' do
+    if params[:content] != nil && params[:content] != ""
+      user = User.find_by(:id => session[:user_id])
+      tweet = Tweet.create(:content => params[:content], :user_id => user.id)
```

Comment:
The tweet creation and user association can both be accomplished in the same step:

``` ruby
@tweet = Tweet.create(content: params[:content], user: current_user)
```

or alternatively

```
@tweet = current_user.tweets.create(content: params[:content])
```